live-build/misc/ansible-roles/appliance-build.virtualization-development/tasks/main.yml

```diff
@@ -43,6 +43,21 @@
       [Service]
       Environment=DLPX_DEBUG=true
 
+- file:
+    path: "/etc/systemd/system/delphix-osadmin.service.d"
+    owner: root
+    group: root
+    state: directory
+    recurse: yes
+
+- copy:
+    dest: "/etc/systemd/system/delphix-osadmin.service.d/override.conf"
+    owner: root
+    group: root
+    content: |
+      [Service]
+      Environment=DLPX_DEBUG=true
+
```

Comment:
I think you can combine each of these with the two above them by using the `with_items` construct. You can find examples of that in the other ansible tasks throughout the codebase.

Reply:
this is consistent with the two tasks above for the mgmt service, so I think this is fine.. this is two different tasks too, `file` and `copy`, so idk if we can combine them like you're thinking..

Reply:
Thanks Seb for pointing out `with_items`! Just updated. 

@prakashsurya Github doesn't allow me to reply your comment directly, so put the reply here. sorry for the confusion. I think what Seb meant is merging `file` and `copy` tasks of three similar systemd services (i.e. delphix-mgmt, delphix-osadmin, delphix-sso-app) into one block using with_items, and the block still has both tasks, but it can iterate all systemd services.